packages/rest/src/providers/reject.provider.ts

```diff
@@ -6,15 +6,19 @@
 import {LogError, Reject, HandlerContext} from '../types';
 import {inject, Provider} from '@loopback/context';
 import {HttpError} from 'http-errors';
-import {writeErrorToResponse} from '../writer';
 import {RestBindings} from '../keys';
+import * as errorHandler from 'strong-error-handler';
 
-const debug = require('debug')('loopback:rest:reject');
+export const defaultErrorHandlerOptions: errorHandler.ErrorHandlerOptions = {
+  log: false,
```

Comment:
Do we still need `log` considering that `writeErrorToResponse` is never logging anything?

Reply:
Oops, can't believe I missed that